src/script/standard.h

```diff
@@ -146,7 +146,7 @@ typedef boost::variant<CNoDestination, PKHash, ScriptHash, WitnessV0ScriptHash,
 bool IsValidDestination(const CTxDestination& dest);
 
 /** Get the name of a txnouttype as a C string, or nullptr if unknown. */
-const char* GetTxnOutputType(txnouttype t);
+std::string GetTxnOutputType(txnouttype t);
```

Comment:
`#include <string>`

Reply:
I've purposely only put ```#include <string>``` where the compiler yells at me since there are other ```.h``` and ```.cpp``` files that use ```std::string``` but omit ```#include <string>``` since there were some other file that had it.

Not quite sure which I should follow...

Reply:
You should include whatever you use in that module. There are certainly existing violations in the codebase, but that's not guidance.

Reply:
Thanks for the input. Added #include <string>